### ISSUE TITLE
aws-efs-csi-driver's helm chart is not working as sub chart

### ISSUE THREAD

**Issue by singh-ajeet** [kubernetes-sigs/aws-efs-csi-driver#225]
My application is using EFS persistent volume, so I was trying to use aws-efs-csi-driver as sub-chart of my application's chart. But it is not working.

There is no error,  but my application is not able to mount EFS. If I deployed aws-efs-csi-driver helm chart separately than it works fine.

Chart.yaml
```
  - name: prometheus
    version: "11.7.*"
    repository: https://kubernetes-charts.storage.googleapis.com/
  - name: nginx-ingress
    version:  "1.40.*"
    repository: https://kubernetes-charts.storage.googleapis.com/
  - name: prometheus-adapter
    version:  "2.4.*"
    repository: https://kubernetes-charts.storage.googleapis.com
  - name: aws-efs-csi-driver
    version:  "0.1.*"
    repository: https://kubernetes-sigs.github.io/aws-efs-csi-driver/
```

After executing ` helm dependency build `  command I can see it in my charts directory as `helm-chart.tgz`.  But it is not getting deployed.

It is working this way - 

```
helm install aws-efs-csi-driver aws-efs-csi-driver/aws-efs-csi-driver
helm install myApplication .
```

**Comment by s3than** [comment]
It's because the helm chart is packaged as helm/ if you add an the following

```
  - name: aws-efs-csi-driver
    version:  "0.1.*"
    repository: https://kubernetes-sigs.github.io/aws-efs-csi-driver/
    alias: helm
```

**Comment by krmichel** [comment]
This should be fixed when PR #291 is merged
